src/applications/appeals/10182/components/IssueCard.jsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { format } from 'date-fns';
+
+import { SELECTED } from '../constants';
+
+/**
+ * IssueCardContent
+ * @param {String} description - contestable issue description
+ * @param {String} ratingIssuePercentNumber - rating %, number with no %
+ * @param {String} approxDecisionDate - date formatted as "YYYY-MM-DD"
+ * @return {React Component}
+ */
+export const IssueCardContent = ({
+  description,
+  ratingIssuePercentNumber,
+  approxDecisionDate,
+}) => {
+  // May need to throw an error to Sentry if any of these don't exist
+  // A valid rated disability *can* have a rating percentage of 0%
+  const showPercentNumber = (ratingIssuePercentNumber || '') !== '';
+
+  return (
+    <div className="widget-content-wrap">
+      {description && <p className="vads-u-margin-bottom--0">{description}</p>}
+      {showPercentNumber && (
+        <p className="vads-u-margin-bottom--0">
+          Current rating: <strong>{ratingIssuePercentNumber}%</strong>
+        </p>
+      )}
+      {approxDecisionDate && (
+        <p>
+          Decision date:{' '}
+          <strong>
+            {format(new Date(`${approxDecisionDate} 00:00:00`), 'MMMM d, yyyy')}
+          </strong>
+        </p>
+      )}
+    </div>
+  );
+};
+
+/**
+ * ContestableIssue
+ * @typedef {Object}
+ * @property {String} ratingIssueSubjectText - contestable issue title
+ * @property {String} description - contestable issue description
+ * @property {String} ratingIssuePercentNumber - rating %, number with no %
+ * @property {String} approxDecisionDate - date formatted as "YYYY-MM-DD"
+ */
+/**
+ * AdditionalIssue
+ * @type {Object}
+ * @property {String} condition - user entered issue name
+ * @property {String} approxDecisionDate - user entered decision date
+ */
+/**
```

Comment:
Been loving the documentation I've seen in your PRs lately. Gotta start doing more of this myself!